lib/matplotlib/tests/test_collections.py

```diff
@@ -309,10 +309,9 @@ def test_add_collection():
     # GitHub issue #1490, pull #1497.
     plt.figure()
     ax = plt.axes()
-    coll = ax.scatter([0, 1], [0, 1])
-    ax.add_collection(coll)
```

Comment:
This is funny enough looking I'm worried this was intentional to test double adding artists or some such.

Reply:
The comment about what the test does and the name of the test seem to be a bit in disagreement. Not sure if it's intentional, but by the comment, it seems like it shouldn't be there...

Reply:
Actually, looking at the original PR and issue, it's definitely about empty collections, and the duplicate `add_collection` seems to be an error.

Reply:
🐑